readthedocs/source/installation.rst

```diff
@@ -12,6 +12,8 @@ as well. See the instructions for :ref:`boltzmann-codes` and
 
 CCL works on Linux or Mac OS. Windows installation is not supported.
 
+CCL works on Linux or Mac OS. Windows installation is not supported.
```

Comment:
Hmm, not sure what's going on here but there seems to be a line duplicated.

Reply:
fixed